packages/react-components/global-context/src/global-context-selector.ts

```diff
@@ -44,11 +43,9 @@ export const createContext = <T>(defaultValue: T, name: string, packageName: str
   // Object symbol properties can't be iterated with `for` or `Object.keys`
   const globalSymbols = Object.getOwnPropertySymbols(globalObject);
   if (!globalSymbols.includes(sym)) {
-    // @ts-expect-error - Indexing object with symbols not supported until TS 4.4
     // eslint-disable-next-line @fluentui/no-context-default-value
-    globalObject[sym] = baseCreateContext(defaultValue);
+    globalObject[sym] = baseCreateContext<unknown>(defaultValue);
```

Comment:
generic without default value is set to `unknown`  type, after bump this was exposed here and triggered errors